### DISCUSSION TITLE
Physics properties of the detectors

### DISCUSSION THREAD

**Discussion by cjnqt** [ozel/DIY_particle_detector#5] [Q&A] [answered]
this is a cool application!

What causes the upper energy range detection limit of ~8 MeV? Is it caused by the amplification range? The diode itself?

**Comment by ozel** [comment, marked as answer]
the diode itself can only provide measurable ionization within its depleted space-charge region (which is proportional to a volume since the sensitive area remains constant). this is a function of the applied bias voltage (see figure 3b in https://www.mdpi.com/1424-8220/19/19/4264/htm) and of course geometrical/physical doping limitations of the diode chip. Ref. 20 of the article suggests that the maximum depleted depth would be somewhere below 100 micrometers.

I calibrated the detector with some low-energy artificial radionuclides and then tested it with natural alpha emitters which go up to about 8.8 MeV max. (polonium-212) - c.f. the article. So for that energy range, it is proven that the energy calibration of the amplifier circuit is correct and linear as described. I could be as well linear for higher energies (and it would work in principle if the bias voltage is increased somewhat) but then you would need to stick it into a particle accelerator beamline, so it is not really relevant for measuring natural radioactivity and neither for artificially created fission products of nuclear reactors, those alpha-emitting isotopes are all within the 8/9 MeV upper range value. 

Another aspect is the input voltage range of the soundcard, larger signals than the ones I tested could saturate them depending on specific soundcard properties. An oscilloscope would not have that limitation but usually has a lower amplitude (y-axis) resolution than the 16-bit of a soundcard/laptop/mobile microphone input.
